geoportal/c2cgeoportal_geoportal/views/raster.py

```diff
@@ -181,6 +196,29 @@ def get_index(index_: int) -> tuple[int, int]:
 
         return result
 
+    def _get_service_data(
+        self, layer: str, lat: float, lon: float, rasters: dict[str, Any]
+    ) -> dict[str, Any] | None:
+        request = f"{rasters[layer]['url']}/height?easting={lon}&northing={lat}"
+        _LOG.info("Doing height request to %s", request)
+        response = requests.get(request, timeout=10)
+        if not response.ok:
+            _LOG.warning("Elevation request %s failed with status code %s", request, response.status_code)
+            raise HTTPInternalServerError(
+                f"Failed to fetch elevation data from {request}: \
```

Comment:
For me, request shouldn't be public.
```suggestion
                f"Failed to fetch elevation data from internal request: \
```